src/ConfigurationRemotingServer/Program.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.Management.Configuration.Processor;
+using System.Runtime.InteropServices;
+using WinRT;

```

Comment:
I know you'll hate me, but can you add stylecops :D

Reply:
😡